### PR TITLE
Add RHEL 8 option to Vagrant VM

### LINKED ISSUE #71: Add RHEL 8 option to Vagrant VM

### PR DESCRIPTION
**Pull request by Dostonbek1**
closes #71

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -26,4 +26,11 @@ Vagrant.configure("2") do |config|
     centos7.vm.network "forwarded_port", guest: 9443, host: 8753
     centos7.vm.synced_folder "./test/centos7/", "/quipucords_installer", :mount_options => ["dmode=777", "fmode=777"]
   end
+
+  config.vm.define "vrhel8" do |rhel8|
+    rhel8.vm.box = "generic/rhel8"
+    rhel8.vm.hostname = "vrhel8"
```
Comment (myersCody) [inline Vagrantfile]: here

Comment (kholdaway) [reply Vagrantfile]: Love this line

File: vm_helper_files/scripts/setup_repos.sh [hunk 2]
```diff
@@ -22,7 +24,12 @@ then
     if [[ $rpm_version == "el6" ]]
     then
         sudo cp -rf config/rhel6/*.repo /etc/yum.repos.d/
-    else
+    elif [[ $rpm_version == "el7" ]]
+    then
         sudo cp -rf config/rhel7/*.repo /etc/yum.repos.d/
+    else
+        sudo cp -rf config/rhel8/*.repo /etc/yum.repos.d/
+        sudo subscription-manager register
```
Comment (myersCody) [inline vm_helper_files/scripts/setup_repos.sh]: you don't need to register and attach

**Comment by Dostonbek1** [comment]
This issue is resolved in this PR: https://github.com/quipucords/quipucords-installer/pull/79

**Pull request closed without merging**